include/aspect/particle/interpolator/interface.h

```diff
@@ -74,6 +75,36 @@ namespace aspect
           std::vector<std::vector<double> >
           properties_at_points(const std::multimap<types::LevelInd, Particle<dim> > &particles,
                                const std::vector<Point<dim> > &positions,
+                               const typename parallel::distributed::Triangulation<dim>::active_cell_iterator &cell = typename parallel::distributed::Triangulation<dim>::active_cell_iterator()) const DEAL_II_DEPRECATED;
+
+          /**
+           * Perform an interpolation of the properties of the particles in
+           * this cell onto a vector of positions in this cell.
+           * Implementations of this function must return a vector of a vector
+           * of doubles with as many entries as positions in @p positions.
+           * Each entry is a vector with as many entries as there are  particle
+           * properties in this computation.
+           * All in @p selected_properties selected components
+           * will be filled with computed properties, all other components
+           * are not filled (or filled with signalling_nan's).
```

Comment:
just called it "invalid values". that it's signalling nans is sort of an implementation detail that not everyone will understand.